drivers/led/lp5569.c

```diff
@@ -83,6 +87,23 @@ static int lp5569_enable(const struct device *dev)
 		return -ENODEV;
 	}
 
+	/* flip the enable pin if specified */
+	if (config->en_pin.port) {
+		if (!gpio_is_ready_dt(&config->en_pin)) {
+			LOG_ERR("Enable gpio not ready");
+			return -ENODEV;
+		}
+
+		ret = gpio_pin_configure_dt(&config->en_pin, GPIO_OUTPUT_HIGH);
```

Comment:
The controller itself requires high, but the hardware might have an active low control signal. 
```suggestion
		ret = gpio_pin_configure_dt(&config->en_pin, GPIO_OUTPUT_ACTIVE);
```

Reply:
Good catch. Active was my intention!